internal/errors/lb.go

```diff
@@ -20,6 +20,9 @@ package errors
 var (
 	ErrEmptySearchResult = New("search result is empty")
 
+	// ErrNotEnoughSearchResult represents an error that the search result is not enough.
+	ErrNotEnoughSearchResult = New("search result is not enough")
```

Comment:
```suggestion
	ErrLessSearchResult = New("search result is less than required")
```

Also, just my idea.
`ErrNotEngouhSearchResult` is fine.

Reply:
Thank you for your suggestion! 
I thought `ErrLessSearchResult ` was more clear!
I have fixed: https://github.com/vdaas/vald/pull/1546/commits/d02435bec16e72f5198da5fd40907de425e99435